go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/th2-net/th2-grpc-common-go
+
+go 1.23.0
```

Comment:
This version is greater than we use for our projects. We won't be able to use this package. I suggest we use the latest version for go1.21